ProtoGame2D/Code/Game/Framework/Game.cpp

```diff
@@ -1,15 +1,58 @@
 #include "Game.h"
 
+#include "Engine/Core/GameCommon.h"
+#include "Engine/Graphics/Renderer.h"
+#include "Engine/Graphics/Shader.h"
+#include "Engine/Graphics/ShaderDefinition.h"
+#include "Engine/Graphics/Buffers/FrameBuffer.h"
+
 Game::Game()
 {
+	m_ShaderDef = new ShaderDefinition(*ShaderDefinition::InitializeDef(APEX_SHADER_XML));
+	m_Shader = Renderer::GetInstance()->GetOrCreateShader(m_ShaderDef);
+	
+	m_ScreenShaderDef = new ShaderDefinition(*ShaderDefinition::InitializeDef(APEX_SCREENSHADER_XML));
+	m_ScreenShader = Renderer::GetInstance()->GetOrCreateShader(m_ScreenShaderDef);
+	
+	Renderer::GetInstance()->EnableBlend(ParseBlendFac[m_Shader->GetSRC()], ParseBlendFac[m_Shader->GetDST()], ParseBlendOp[m_Shader->GetOP()]);
+
+	m_CurrentBuffer = new FrameBuffer();
+	m_NextBuffer = new FrameBuffer();
```

Comment:
Why the double buffer?

Reply:
Bhaiya, one is meant to be the source buffer and the other is meant to be destination buffer.